Packages/io.chainsafe.web3-unity/Runtime/Scripts/Model/MarketplaceModel.cs

```diff
@@ -1,59 +1,171 @@
 ﻿using System.Collections.Generic;
+using Newtonsoft.Json;
 using BigInteger = System.Numerics.BigInteger;
 
 namespace ChainSafe.Gaming.UnityPackage.Model
 {
     public class MarketplaceModel
     {
-        public class Root
-        {
-            public int page_number { get; set; }
-            public int page_size { get; set; }
-            public int total { get; set; }
-            public string cursor { get; set; }
-            public List<Item> items { get; set; }
-            public List<Owners> owners { get; set; }
+        #region ProjectMarketplaces
```

Comment:
This should be moved to Samples too (if you still need it), otherwise, we are going to have two implementations of the same thing in the package.

Reply:
Just to be clear, I'm talking about the whole implementation, not just this file.